README.md

```diff
@@ -25,6 +25,19 @@ make distclean
 ```
 
 ## Usage
+ 
+- If the tile is valid, `vtvalidate` will return an empty string. 
+- If the tile is invalid, `vtvalidate` will return the string output from vtzero. 
+- `vtvalidate` will throw an error if there is unexpected behaviour, for example an invalid parameter value or a corrupt compressed buffer.
```

Comment:
@GretaCB what is a "parameter value" in this case?

Reply:
Fixed!